### PR TITLE
origin detection over UDP

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do?
adds instructions for origin detection over UDP

### Motivation
@hkaj 

### Preview link

https://docs-staging.datadoghq.com/cswatt/origin-detection-udp/agent/kubernetes/dogstatsd/#origin-detection-over-udp

### CODE REVIEW

**Comment by hkaj** [comment]
i think we should also mention the link with DD_DOGSTATSD_CARDINALITY, as it changes how many tags we'll add to metrics with origin detection. And it also impacts billing (because more precise tagging --> more metrics), so a warning about this would be nice.

**Comment by irabinovitch** [comment]
@cswatt can we add minimum version numbers of the agent and client libraries here?

**Comment by cswatt** [comment]
@hkaj how does this look now?

**Comment by hkaj** [comment]
ping - where are we with this?

File: content/en/agent/kubernetes/dogstatsd.md [hunk 1]
```diff
@@ -100,19 +100,50 @@ env:
 
 With this, any pod running your application is able to send DogStatsD metrics via port `8125` on `$DOGSTATSD_HOST_IP`.
 
+## Origin detection over UDP
+
+Origin detection allows DogStatsD to detect where the container metrics come from, and tag metrics automatically. When this mode is enabled, all metrics received via UDP are tagged by the same container tags as Autodiscovery metrics.
+
+**Note**: as an alternative to UDP, you may also use [Unix Domain Sockets][18].
```
Comment (ruthnaebeck) [inline content/en/agent/kubernetes/dogstatsd.md]:
```suggestion
**Note**: An alternative to UDP is [Unix Domain Sockets][18].
```

**Comment by ruthnaebeck** [comment]
One wording suggestion, otherwise 👍

**Pull request merged**